test/remote/too_many_verify_codes.js

```diff
@@ -22,54 +21,50 @@ process.env.RATE_LIMIT_INTERVAL_SECONDS = 1
 process.env.IP_RATE_LIMIT_INTERVAL_SECONDS = 1
 process.env.IP_RATE_LIMIT_BAN_DURATION_SECONDS = 1
 
-var testServer = new TestServer(config)
+const testServer = new TestServer(config)
 
-var client = restify.createJsonClient({
+const client = restify.createJsonClient({
   url: 'http://127.0.0.1:' + config.listen.port
 })
 
-Promise.promisifyAll(client, { multiArgs: true })
+Promise.promisifyAll(client, {multiArgs: true})
 
-test(
-  'startup',
-  function (t) {
-    testServer.start(function (err) {
-      t.type(testServer.server, 'object', 'test server was started')
+test('startup', (t) => {
+  testServer.start((err) => {
+    t.type(testServer.server, 'object', 'test server was started')
+    t.notOk(err, 'no errors were returned')
+    t.end()
+  })
+})
+
+const testCasesVerifyCode = [
```

Comment:
style nit: I wonder if this would be clearer as `const VERIFY_CODE_ACTIONS` or similar